Sources/OTT/KalturaOTTPlayer.swift

```diff
@@ -181,9 +181,7 @@ import PlayKitKava
             let ovpEntryId = mediaEntry.metadata?["entryId"] ?? options.assetId ?? ""
             self.updateKavaPlugin(ovpPartnerId: ovpPartnerId, ovpEntryId: ovpEntryId, mediaOptions: options)
             self.updatePhoenixAnalyticsPlugin()
-            
-            self.mediaEntry = mediaEntry
-            callback(nil)
```

Comment:
This is needed, after the 'updateMediaEntryWithLoadedInterceptors'

Reply:
If we need keep `callback(nil)` here, then we have to return callback to `self.updateMediaEntryWithLoadedInterceptors(mediaEntry)`
According to [sample  App and implementation](https://github.com/kaltura/kaltura-player-ios-samples/blob/5479f940553a4fb7d1b403149f10d634b508adaf/OTTSample/OTTSample/PlayerViewController/MediaPlayerViewController.swift#L152) of `kalturaOTTPlayer.loadMedia(options: mediaOptions) { [weak self] (error) in` in it, once we go to this closure with `error==nil` this code at the end will performed.
```
DispatchQueue.main.async {
                        if !videoData.player.autoPlay {
                            self.activityIndicator.stopAnimating()
                        }
                    }
```


so if we add just like this
```
self.updateMediaEntryWithLoadedInterceptors(mediaEntry)
callback(nil)
```
it is going to be an issue when for example `self.activityIndicator.stopAnimating()` already called but Interceptors logic still in progress.

In previous iteration this part was like this
```
self.updateMediaEntryWithLoadedInterceptors(mediaEntry) { (error: Error?) in
                callback(error)
            }
```
Then we changed it to current. So I would like to change it one more time. And add Void closure to `updateMediaEntryWithLoadedInterceptors`  It would look like next:

`self.updateMediaEntryWithLoadedInterceptors(mediaEntry) { callback(nil) }`

Reply:
Of course, not just to add the callback back, but after the 'updateMediaEntryWithLoadedInterceptors' is done.